biz.aQute.bndlib.tests/test/test/servicecapability/ServiceCapabilityTest.java

```diff
@@ -0,0 +1,98 @@
+package test.servicecapability;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.osgi.namespace.service.ServiceNamespace;
+
+import aQute.bnd.annotation.service.ServiceCapability;
+import aQute.bnd.header.Attrs;
+import aQute.bnd.header.Parameters;
+import aQute.bnd.osgi.Builder;
+import aQute.bnd.osgi.Domain;
+import aQute.lib.io.IO;
+import test.servicecapability.misc.Foo;
+import test.servicecapability.misc.Foo2;
+import test.servicecapability.misc.Foo3;
+import test.servicecapability.misc2.Bar;
+
+@ServiceCapability(value = ServiceCapabilityTest.class)
+@ServiceCapability(value = Foo.class, attribute = "test=test")
+@ServiceCapability(value = Foo2.class, uses = Bar.class)
+@ServiceCapability(value = Foo3.class, uses = {
+	Bar.class, ServiceCapabilityTest.class
+})
+public class ServiceCapabilityTest {
+
+
+	@Test
+	public void testServiceCapability() throws Exception {
+		Attrs attrs = assertCapability(ServiceCapabilityTest.class);
+		assertEquals("test.servicecapability", attrs.get("uses:"));
```

Comment:
```suggestion
		assertThat(attrs).containsEntry("uses:" , "test.servicecapability");
```

It would be nice to use AssertJ testing for better readability and better error output.